benchmarks/nemo/tracer_advection/multi_kernel/Makefile

```diff
@@ -0,0 +1,87 @@
+# Makefile for the version of the tra_adv benchmark that has had
```

Comment:
I think there should be a README in this directory, as you have for the original version.

Reply:
Yes, agreed and added.